### PR TITLE
OPENEUROPA-680: Add paragraph step to test.

### PR DESCRIPTION
**Pull request by imanoleguskiza**
Add paragraph step to test.

### CODE REVIEW

File: features/landing-page.feature [hunk 1]
```diff
@@ -10,17 +10,22 @@ Feature: Landing page
...
     And I press "Save"
     Then I should see "Landing page Homepage has been created." in the "message"
     And I should see "Homepage" in the "page header"
     And I should see "This is the home page of the site" in the "page header"
+    And I should see "This is the content of the page." in the "content"
 
     # Translate the Landing page content into Spanish.
     When I click "Translate"
     And I click "Add" in the "Spanish" row
     And I fill in "Título" with "Página principal"
```
Comment (Medi4tor) [inline features/landing-page.feature]: Labels should be in English.

File: features/landing-page.feature [hunk 2]
```diff
@@ -10,17 +10,22 @@ Feature: Landing page
...
     And I fill in "Introduction" with "Esta es la página principal del sitio"
+    And I fill in "Texto" with "Este es el contenido de la pagina."
```
Comment (Medi4tor) [inline features/landing-page.feature]: idem

File: features/landing-page.feature [hunk 3]
```diff
@@ -10,17 +10,22 @@ Feature: Landing page
...
     And I press "Guardar (this translation)"
     Then I should see "Landing page Página principal ha sido actualizado." in the "message"
```
Comment (Medi4tor) [inline features/landing-page.feature]: We should use the drupal command about success message.

**Comment by netlooker** [comment]
I have rebased the branch to align with 'master'.

**Pull request merged**